alpine/build.sh

```diff
@@ -1,18 +1,18 @@
 #!/bin/bash
-if [[ $# -eq 0 ]]; then
-  echo "This script needs version as argument"
-  echo "example : ./build.sh 3.8"
-  exit -1
-fi
+base_image_version='3.12'
 
-readonly alpine_baseurl=http://dl-cdn.alpinelinux.org/alpine/
-readonly arch_url=${alpine_baseurl}v${1}/releases/x86_64/
-curl -OL ${arch_url}latest-releases.yaml
-readonly filename=`cat latest-releases.yaml | grep file | grep minirootfs | awk '{print $2}'`
-curl -OL ${arch_url}${filename}
+while getopts b:h option
+do
+	case "${option}"
+		in
```

Comment:
looks like it is 4 spacing indentation, can we use 2?

Reply:
update editor config to use space indented with 2